### PR TITLE
adding nodemailer structure and configuring arquitecture

### PR DESCRIPTION
**Pull request by Strike2-ux**
I did have problems with the structure of the Architecture, but what I add to the code is not too far from reality, we need nodemailer and declare the api/auth/recovery & install the utils, please check it if am i wrong (probably), keep foward to the development.

### CODE REVIEW

**Comment by ViloriaUrdaneta** [comment]
> I did have problems with the structure of the Architecture, but what I add to the code is not too far from reality, we need nodemailer and declare the api/auth/recovery & install the utils, please check it if am i wrong (probably), keep foward to the development.

I'm a bit confused by the change of architecture because I was already working on the basis of the previous one.
I think token.utils.js is the same as jwt.js.

File: server.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import express from 'express';
```
Comment (AlfredoVillegas) [inline server.js:1]: puedes ayudarme a saber cual seria la intencion de este archivo ?. 
ya estamos estructurando la app y levantandola en el index.js del nivel superior en la estructura de carpetas

File: server.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  })
+  .catch((err) => {
+    console.log(err.message);
+  });
+*/
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
```
Comment (AlfredoVillegas) [inline server.js:23]: creo que son rutas que no manejamos

File: src/models/index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+'use strict'
```
Comment (AlfredoVillegas) [inline src/models/index.js:1]: ya hay una configuracion que establece squilize, esta dento de stc/database/sequelize.js

**Review by AlfredoVillegas — CHANGES_REQUESTED** [review]
creo que hay archivos que no iban en este commit, y los añadiste por error. pudes revisarlo nuevamente?

**Pull request closed without merging**